test/TestProjects/LRO-LowLevel/Generated/LROServiceClient.cs

```diff
@@ -0,0 +1,520 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+// <auto-generated/>
+
+#nullable disable
+
+using System;
+using System.Collections.Generic;
+using System.Runtime.CompilerServices;
+using System.Threading;
+using System.Threading.Tasks;
+using Azure;
+using Azure.Core;
+using Azure.Core.Pipeline;
+
+namespace LRO_LowLevel
+{
+    /// <summary> The LROService service client. </summary>
+    public partial class LROServiceClient
```

Comment:
Here is the Generated test project which covers Short LRO, Long LRO, Pageable Short LRO, Pageable Long LRO